samples/v1.3/Elements/Input.Text.InlineAction.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
We decided not to publicly show the updated Element cards until most of our hosts move to v1.3, so this needs to be renamed or put in a different folder so it won't be surfaced until we are ready.

Reply:
Which is tracked by #4612